### PR TITLE
feat(failure): add failure in relevant doc location

### PR DESCRIPTION
**Pull request by rbioteau**
Closes BPM-330

### CODE REVIEW

File: modules/runtime/pages/admin-application-task-list.adoc [hunk 1]
```diff
@@ -66,6 +66,16 @@ This means that the flow node does not have to be done successfully for the foll
 
 The task is moved from "Failed" to "Done", while the connector stays in the "Failed" state.
 
+
+== Error details
+
+This feature is available with the Access, Enterprise or the Performance editions.
```
Comment (abirembaut) [inline modules/runtime/pages/admin-application-task-list.adoc]:
```suggestion
This feature is available with the Access, Enterprise, Efficiency or Performance editions.
```

File: modules/runtime/pages/admin-application-task-list.adoc [hunk 2]
```diff
@@ -66,6 +66,16 @@ This means that the flow node does not have to be done successfully for the foll
...
+
+When a flownode is failed, an additional "Error details" panel displays the information about the exception that caused the failure.
+When available, the scope and context of the error is attached to failure to help locating the issue in the process definition.
```
Comment (abirembaut) [inline modules/runtime/pages/admin-application-task-list.adoc]:
```suggestion
When available, the scope and context of the error is attached to the failure to help locating the issue in the process definition.
```

File: modules/runtime/pages/work-execution.adoc [hunk 3]
```diff
@@ -55,6 +55,10 @@ When a work fails, there is a mechanism in place to retry it. If we identify tha
 * set a date before which the work should not be retried
 * increment the `retryCount` of the `WorkDescriptor`
 
+=== Failures
+
+When a flownode fails, the exception that caused the failure is stored and can be retrieved by the API (Enterprise only). A failure is annotated with additional context information and can also be consulted after the flownode has been archived.
```
Comment (abirembaut) [inline modules/runtime/pages/work-execution.adoc]:
```suggestion
When a flownode fails, the exception that caused the failure is stored and can be retrieved by the API (subscription editions only). A failure is annotated with additional context information and can also be consulted after the flownode has been archived.
```

**Review by abirembaut — COMMENTED** [review]
Just a few suggestions

**Review by techosbonitasoft — APPROVED** [review]

**Pull request merged**